Makefile

```
@@ -17,6 +18,7 @@ FORMATTED_AREAS=\
 	tests/test_helpers.py \
 	tests/test_protocol.py \
 	tests/test_protocol_object_conversion.py \
+	tests/coordinator \
```

_nit:_ Let's be consistent
```suggestion
	tests/coordinator/ \
```